libs/designsystem/src/lib/components/empty-state/empty-state.component.ts

```diff
@@ -1,13 +1,39 @@
-import { Component, Input } from '@angular/core';
+import { AfterContentInit, Component, ContentChildren, Input, QueryList } from '@angular/core';
+import { delay } from 'rxjs/operators';
+
+import { ButtonComponent } from '../button/button.component';
 
 @Component({
   selector: 'kirby-empty-state',
   templateUrl: './empty-state.component.html',
   styleUrls: ['./empty-state.component.scss'],
 })
-export class EmptyStateComponent {
+export class EmptyStateComponent implements AfterContentInit {
   @Input() iconName: string;
   @Input() customIconName: string;
   @Input() title: string;
   @Input() subtitle: string;
+
+  @ContentChildren(ButtonComponent)
+  private slottedButtons: QueryList<ButtonComponent>;
+
+  ngAfterContentInit() {
+    this.enforceAttentionLevelRules();
+
+    this.slottedButtons.changes.pipe(delay(0)).subscribe(() => this.enforceAttentionLevelRules());
```

Comment:
Why the need for `delay`? Consider adding comment...

Reply:
Is this sufficient 😄?

Reply:
Yes much better :)
Consider stating why it's necessary as we do in RadioGroupComponent:
`// setTimeout prevents ExpressionChangedAfterItHasBeenCheckedError when updating the DOM in QueryList.changes:`
You could also consider wrapping the call to `this.enforceAttentionLevelRules` in a timeout instead, as this is the conventional way to postpone a state change to next CD...
See: https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts#L201

Reply:
I've updated the comment to be: 
`/* setTimeout prevents ExpressionChangedAfterItHasBeenCheckedError when changing attention 
     levels of slotted buttons in this.enforceAttentionLevelRules */`

and removed the use of delay 😀